examples/addons/azure-policy/README.md

```diff
@@ -93,12 +93,25 @@ kubectl get pods -n kube-system
 | Name           | Required | Description                       | Default Value                                        |
 | -------------- | -------- | --------------------------------- | ---------------------------------------------------- |
 | name           | no       | container name                    | "gatekeeper"                                         |
-| image          | no       | image                             | "quay.io/open-policy-agent/gatekeeper:v3.0.4-beta.2" |
+| image          | no       | image                             | "mcr.microsoft.com/oss/open-policy-agent/gatekeepe:v3.1.0-beta.7" |
 | cpuRequests    | no       | cpu requests for the container    | "100m"                                               |
 | memoryRequests | no       | memory requests for the container | "256Mi"                                              |
-| cpuLimits      | no       | cpu limits for the container      | "100m"                                               |
+| cpuLimits      | no       | cpu limits for the container      | "1000m"                                              |
 | memoryLimits   | no       | memory limits for the container   | "512Mi"                                              |
 
+## Disable Azure Policy Add-on
+
+### Option 1
```

Comment:
@sozercan You mentioned user has to clean up constraint templates and constraints also on uninstall. Should we add that step too?

Reply:
afaik you dont need to cleanup gk created resource, when user reenables, it should apply on top of existing ones if they chose not to cleanup manually